testdata/bundles/registry-v1/prometheus-operator.v0.47.0/metadata/annotations.yaml

```diff
@@ -1,4 +1,5 @@
 annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: beta
```

Comment:
Why is this being added back? https://github.com/operator-framework/operator-controller/pull/250#discussion_r1214652172